storm-core/src/clj/org/apache/storm/daemon/acker.clj

```diff
@@ -18,74 +18,24 @@
   (:import [org.apache.storm.tuple Tuple Fields])
```

Comment:
Lets just delete this file entirely.  There is not reason to play the games with wrapping the acker bolt and using the Container so that the acker can be serializable.  AckerBolt is already serializable and has the ID set properly so lets just delete this file.